### ISSUE TITLE
Reference resized bureau logo images

### ISSUE THREAD

**Issue by timwood** [CommerceGov/DataCorps_WebSite#12]
Follow up on https://github.com/CommerceGov/DataCorps_WebSite/issues/4

Some of the bureau logo images are still referencing the full size images and others are referencing the 200x200 image, yet all are resized with CSS to 75 pixels wide. It's a waste of bandwidth and page load speed to load images that are bigger than they will appear on the page, especially on mobile. I recommend using the 100x100 image style/size for each logo as that is the closest automatic resizing we have defined in Drupal and which the images are available. You should be able to reference the 100x100 image style by changing the url from (BEA example):

/sites/commerce.gov/files/media/images/2015/bea_logo_0.jpg
TO
/sites/commerce.gov/files/styles/scale_100x100/public/media/images/2015/bea_logo_0.jpg

So basically adding "styles/scale_100x100/public/" to the existing paths, after "files/".

In the BEA example the image size loaded would be reduced from 250KB to 7.8KB, a significant savings.

**Comment by SigmaMonstR** [comment]
​What's the link for the ITA100x100. I can get the 200x200, but for the
smaller version, I can't seem to get the URL right.​

On Wed, Sep 16, 2015 at 10:39 AM, Tim Wood <notifications@github.com> wrote:

> Follow up on #4
> <https://github.com/CommerceGov/DataCorps_WebSite/issues/4>
>
> Some of the bureau logo images are still referencing the full size images
> and others are referencing the 200x200 image, yet all are resized with CSS
> to 75 pixels wide. It's a waste of bandwidth and page load speed to load
> images that are bigger than they will appear on the page, especially on
> mobile. I recommend using the 100x100 image style/size for each logo as
> that is the closest automatic resizing we have defined in Drupal and which
> the images are available. You should be able to reference the 100x100 image
> style by changing the url from (BEA example):
>
> /sites/commerce.gov/files/media/images/2015/bea_logo_0.jpg
> TO
> /sites/
> commerce.gov/files/styles/scale_100x100/public/media/images/2015/bea_logo_0.jpg
>
> So basically adding "styles/scale_100x100/public/" to the existing paths,
> after "files/".
>
> In the BEA example the image size loaded would be reduced from 250KB to
> 7.8KB, a significant savings.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CommerceGov/DataCorps_WebSite/issues/12>.
>



-- 
*Jeff Chen *
jeffchen.org
contact@jeffchen.org
914-837-1336

**Comment by timwood** [comment]
Pull request should have all correct paths, although I didn't test in detail.

The URL for ITA would be https://www.commerce.gov/sites/commerce.gov/files/styles/scale_100x100/public/media/images/branding/ita200x200px.jpg

**Comment by SigmaMonstR** [comment]
Committed. Note that most of the Data Corps  can't access anything on
home.commerce.gov.

On Wed, Sep 16, 2015 at 10:50 AM, Tim Wood <notifications@github.com> wrote:

> Pull request should have all correct paths, although I didn't test in
> detail.
>
> The URL for ITA would be
> https://www.commerce.gov/sites/commerce.gov/files/styles/scale_100x100/public/media/images/branding/ita200x200px.jpg
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CommerceGov/DataCorps_WebSite/issues/12#issuecomment-140764668>
> .
>



-- 
*Jeff Chen *
jeffchen.org
contact@jeffchen.org
914-837-1336

**Comment by timwood** [comment]
Are you committing to another branch or just not pushing to Github yet? Because I'm not seeing any of the commits from today in Github.

Have the Data Corps members worked with the service desk to get proper network access and VPN access so they can access home.commerce.gov and other internal resources?

**Comment by tyronegrandison** [comment]
@timwood there is no internal access right now. Still trying to get approved machines :(

**Comment by SigmaMonstR** [comment]
@timwood i just refreshed and saw it went in directly to the gh-pages branch.

**Comment by tyronegrandison** [comment]
Looks complete.
